### PR TITLE
Stylelint back to package. Some bug after merger with master

### PR DESCRIPTION
**Pull request by ilyafefelov**

### CODE REVIEW

**Comment by ilyafefelov** [comment]
[add stylelint package back package.](https://trello.com/c/A5DieuyM/81-add-stylelint-package-back-package)

File: client/src/App.css [hunk 1]
```diff
@@ -1,3 +1,3 @@
 body {
-    background: #e9967a;
-}
+  background: #e9967a;
```
Comment (zozich) [inline client/src/App.css]: After using sass compiler you should not any .css file in the repo, only `.scss`

**Comment by ilyafefelov** [comment]
check

File: .gitignore [hunk 2]
```diff
@@ -294,3 +293,6 @@ node_modules/
 # Windows shortcuts
 *.lnk
 /client/package-lock.json
+src/Containers/
```
Comment (zozich) [inline .gitignore]: Why would you add that to `.gitignore`?

File: .gitignore [hunk 3]
```diff
@@ -294,3 +293,6 @@ node_modules/
...
+yarn.lock
+src/index.css
```
Comment (zozich) [inline .gitignore]: *.css already added above

File: client/.stylelintcache [hunk 4]
```diff
@@ -1 +1 @@
-{"/Users/illya/Desktop/final.project.danit./TextMessenger/client/src/App.scss":{"size":55,"mtime":1532016642037,"hashOfConfig":"sgvaad"}}
\ No newline at end of file
+{"/Users/illya/Desktop/final.project.danit./TextMessenger/client/src/index.scss":{"size":63,"mtime":1532092826462,"hashOfConfig":"1cnsiqp"}}
```
Comment (zozich) [inline client/.stylelintcache]: Add to `.gitignore`, this file should not appear on GitHub

**Pull request merged**